app/views/giraffe/contribute.scala.html

```diff
@@ -27,10 +30,19 @@ <h2 class="giraffe__heading--sub">
             -->
             </div>
 
-            <div class="form__container">
+            @for(message <- errorMessage) {
+                <div class="error_message" id="errorMessage">
+                        <h2 class="form__column-heading headline-text--bolder">There was a problem</h2>
+                        @message
+                        <a class="action action--contribute js-advance" id="error_tryAgain" >Try again</a>
+                </div >
+            }
+
+            <div class="form__container @containerVisibility">
                 <div class="form__switcher">
                     <a href="#" class="js-switcher-contribute form__container--active" data-switches="js-contribution"></a> <a class="js-switcher-details" href="#" data-switches="js-details"></a> <a class="js-switcher-pay" href="#" data-switches="js-payment"></a>
                 </div>
+                @fragments.giraffe.paypalForm(countryGroup, cmpCode, intCmpCode)
```

Comment:
Nope, there's no need for any of it, we can just rename `contributeReact` to `contribute`